VMD_scripts/Blob_GUI.tcl

```diff
@@ -5,21 +5,45 @@ if [winfo exists .blob] {
 }
 
 namespace eval ::blobulator {
-	variable buttonWidth 54
+	# Widths,Heights,Rows and Columns
+	variable buttonWidth 102
 	variable defaultButtonWidth 7
 	variable dropDownMenuWidth 14 
-	variable isFirst 0
+	variable canvasWidth 640
+	variable canvasHeight 2
+	variable atomselectWidth 24
+	variable paraWidth 10
+	variable sliderRow 5
+	variable sliderLength 420
+	variable checkmarkColumn 4
+	variable textColumn 0
+	variable paraColumn 1
+	variable text2Column 2
+	variable textVariableColumn 3
+	variable dropDownColumn 3
+	variable checkBoxColumn 4
+	variable defaultButtonColumn 4
+	
+	#Variables that are strings
 	variable blobColorType1 "Blob Type"
 	variable blobColorType2 "Blob ID"
 	variable hydropathyScale1 "Kyte-Doolittle"
 	variable hydropathyScale2 "Eisenberg-Weiss"
 	variable hydropathyScale3 "Moon-Fleming"
 	variable graphRepOptions "Blob ID"
+
+
+	#Switch variables and default values 
+	variable hydropathyScaleDictionaryList "Kyte-Doolittle"
```

Comment:
What is this variable for?

Reply:
For GUI's it helps to have default values that are instantly selected, less initial work for the user. This sets the hydrophobicity scale to Kyte-Doolittle on startup.